tests/test_application.py

```diff
@@ -68,7 +68,7 @@
                 "parameters": [
                     {
                         "required": True,
-                        "schema": {"title": "Item_Id"},
+                        "schema": {"title": "Item Id"},
```

Comment:
At some point recently, the title generation logic changed to use spaces instead of underscores.

Reply:
Nice! :sparkles: